### PR TITLE
Fixing some typos

### PR DESCRIPTION
**Pull request by osc135**
Just proofreading and fixing some grammar and other errors!

### CODE REVIEW

**Comment by marccampbell** [comment]
@salahalsaleh adding you and @dexhorthy as reviewers.

File: content/kotsadm/installing/airgap-packages.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Once this message is displayed visit `http://localhost:8800` to complete the app
 ![Airgap Bundle](/images/airgap-install.png)
 
 ### Upload Airgap Bundle
-The software vendor should have delivered a `.airgap` bundle to be used on this screen. The bundle contains the container images and manifests. Choose the bundle and click continue to start processing.
+The software vendor should have delivered a `.airgapped` bundle to be used on this screen. The bundle contains the container images and manifests. Choose the bundle and click continue to start processing.
```
Comment (salahalsaleh) [inline content/kotsadm/installing/airgap-packages.md:59]: this is a file extension, and it is `.airgap` not `.airgapped`

File: content/vendor/entitlements/built-in-entitlements.md [hunk 2]
```diff
@@ -15,11 +15,11 @@ All KOTS applications that are packaged and delivered through the Replicated [Ve
 
 **Description**: If a license has an expiration date, the expiration date will be included in the license here. The date is encoded in ISO 8601 format (e.g., `2026-01-23T00:00:00Z`). If a license does not expire, this field will be missing.
 
-### Airgap Support
+### Airgapped Support
 
-**Field Name**: `isAirgapSupported`
```
Comment (salahalsaleh) [inline content/vendor/entitlements/built-in-entitlements.md:20]: this should be kept as `isAirgapSupported`

File: content/vendor/helm/helm-airgap-builder.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
 ---
 date: 2020-01-03
-linktitle: "Helm Airgap Builder"
-title: Helm Airgap builder
+linktitle: "Helm Airgapped Builder"
```
Comment (salahalsaleh) [inline content/vendor/helm/helm-airgap-builder.md:3]: This sounds wrong. Makes it like the builder is airgapped. Which is not the case. I think it should be kept as `Airgap` (next line as well)

File: content/vendor/helm/helm-airgap-builder.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 
 To solve this problem, the KOTS [HelmChart custom resource](/reference/v1beta1/helmchart/) contains a `builder` attribute for vendors to provide values that will `helm template` to the full manifest.
 
-The values passed in `builder` should be the bare minimum [Helm Values](https://helm.sh/docs/chart_template_guide/values_files/) to be able to template out the Helm chart so that it will have the correct image tags. Upon promotion to a release channel, on the back-end Airgap Builder will, template the chart with the builder values, render the resulting YAML manifests and export any images referenced to build the airgap package. 
+The values passed in `builder` should be the bare minimum [Helm Values](https://helm.sh/docs/chart_template_guide/values_files/) to be able to template out the Helm chart so that it will have the correct image tags. Upon promotion to a release channel, on the back-end Airgapped Builder will, template the chart with the builder values, render the resulting YAML manifests and export any images referenced to build the airgap package.
```
Comment (salahalsaleh) [inline content/vendor/helm/helm-airgap-builder.md:12]: same here, i think it should stay `Airgap`

File: content/vendor/entitlements/built-in-entitlements.md (hunk 2 above)
Comment (osc135) [reply content/vendor/entitlements/built-in-entitlements.md:20]: @salahalsaleh i think i addressed this one, thanks

**Comment by salahalsaleh** [comment]
Thanks for fixing all of this! @osc135

**Pull request merged**